malasakit-django/pcari/admin.py

```diff
@@ -397,13 +398,57 @@ def options(self, option_question):
     search_fields = ('prompt', 'options', 'tag')
 
 
+@admin.register(Location, site=site)
+class LocationAdmin(admin.ModelAdmin):
+    """ Admin behavior for :class:`pcari.models.Location`. """
+    def display_country(self, location):
+        return location.country or self.empty_value_display
+    display_country.short_description = 'Country'
+
+    def display_province(self, location):
+        return location.province or self.empty_value_display
+    display_province.short_description = 'Country'
+
+    def display_municipality(self, location):
+        return location.municipality or self.empty_value_display
+    display_municipality.short_description = 'Country'
+
+    def display_division(self, location):
+        return location.division or self.empty_value_display
+    display_division.short_description = 'Country'
+
+    def enable_as_input(self, request, queryset):
```

Comment:
add description for the action so the admins know how to use it

Reply:
Will rename and add `help_text` for the `enabled` field.